src/main/java/uk/gov/hmcts/ccd/v2/internal/controller/UIDraftsController.java

```diff
@@ -86,7 +84,6 @@ public UIDraftsController(
             caseTypeId));
     }
 
-    @Transactional
     @PutMapping(
         path = "/case-types/{ctid}/drafts/{did}",
         headers = {
```

Comment:
Is this method covered in Transactional scope in server layer or on repository?

Reply:
Yes it covered the implementation of executeUpdate is @Transactional